
package galaxy;

message ResourceItem {
    optional string name = 1;   //Name could  be memory,cpu or port 
    optional int64 value = 2;
}

message TaskStatus{
    optional int64 task_id = 1;
    optional int32 status = 2;  //deploying,starting,running,error
}

message TaskInfo{
    optional int64 task_name = 1;
    optional bytes task_raw = 2;
    optional string cmd_line = 3;
}
